### ISSUE TITLE
TypeError in CLIBot.iter_updates invocation

### ISSUE THREAD

**Issue by coagulant** [pyupio/pyup#191]
`pyup --repo=Eksmo/eksmo --user-token=mytoken --initial` crashed cli (version 0.5.0)

```
Traceback (most recent call last):
  File "/Users/prophet/.envs/mybook/bin/pyup", line 11, in <module>
    sys.exit(main())
  File "/Users/prophet/.envs/mybook/lib/python2.7/site-packages/click/core.py", line 716, in __call__
    return self.main(*args, **kwargs)
  File "/Users/prophet/.envs/mybook/lib/python2.7/site-packages/click/core.py", line 696, in main
    rv = self.invoke(ctx)
  File "/Users/prophet/.envs/mybook/lib/python2.7/site-packages/click/core.py", line 889, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/prophet/.envs/mybook/lib/python2.7/site-packages/click/core.py", line 534, in invoke
    return callback(*args, **kwargs)
  File "/Users/prophet/.envs/mybook/lib/python2.7/site-packages/pyup/cli.py", line 44, in main
    bot.update(branch=branch, initial=initial, pin=pin, close_prs=close_prs)
  File "/Users/prophet/.envs/mybook/lib/python2.7/site-packages/pyup/bot.py", line 94, in update
    scheduled=kwargs.get("scheduled", False)
  File "/Users/prophet/.envs/mybook/lib/python2.7/site-packages/pyup/bot.py", line 117, in apply_updates
    self.iter_updates(initial, scheduled)
TypeError: iter_updates() takes exactly 2 arguments (3 given)
```

**Comment by psykzz** [comment]
Also getting this error if you run it without `--initial`, although the line it occurs moves but its related.

```
Traceback (most recent call last):
  File "/usr/bin/pyup", line 11, in <module>
    sys.exit(main())
  File "/usr/lib/python2.7/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/usr/lib/python2.7/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/usr/lib/python2.7/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/lib/python2.7/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/usr/lib/python2.7/site-packages/pyup/cli.py", line 44, in main
    bot.update(branch=branch, initial=initial, pin=pin, close_prs=close_prs)
  File "/usr/lib/python2.7/site-packages/pyup/bot.py", line 94, in update
    scheduled=kwargs.get("scheduled", False)
  File "/usr/lib/python2.7/site-packages/pyup/bot.py", line 143, in apply_updates
    for title, body, update_branch, updates in self.iter_updates(initial, scheduled):
TypeError: iter_updates() takes exactly 2 arguments (3 given)
```

**Comment by jayfk** [comment]
Sorry for that! The CLI could use some automated tests to avoid stuff like this.

**Comment by jayfk** [comment]
Fixed in `0.6.0`!
